platform/linux-generic/include/odp_schedule_if.h

```diff
@@ -95,6 +95,7 @@ typedef struct {
 				   odp_schedule_group_info_t *);
```

Comment:
Don't know how to comment on COMMIT LOG on Github !

Anyway, commit log needs correct format "linux-gen: sched: ...." and some more text (it's empty now)

Reply:
@psavol only comment like you did.